chapter_linear-regression/synthetic-regression-data.md

```diff
@@ -173,6 +188,15 @@ As before, we need a dataset with features `X` and labels `y`.
 Beyond that, we set `batch_size` in the built-in data loader 
 and let it take care of shuffling examples  efficiently.
 
+:begin_tab:`jax`
+JAX is laser-focused on program transformations and
+accelerator-backed NumPy, so it doesn’t include data loading or munging
```

Comment:
so the current version (v0.XXX) does not include data loading or ...

Reply:
This is straight out of their documentation. See [here](https://jax.readthedocs.io/en/latest/notebooks/Neural_Network_and_Data_Loading.html#data-loading-with-pytorch). I don't think JAX will ever have a dataloader. Even if they do get something, it will be an external package in the JAX Ecosystem.

Hardcoding a version here is also not a wise choice IMO